### DISCUSSION TITLE
Installing Crouton

### DISCUSSION THREAD

**Discussion by rollo29** [dnschneid/crouton#5000] [Q&A]
I have ran into a problem trying to install Croton this is a screet shot of the instructions I was using and a screen shot of the issue I am having please help me???
![Screenshot 2023-07-21 1 25 02 AM](https://github.com/dnschneid/crouton/assets/110459249/a8ef72ff-18d1-4735-a53c-368f0326a006)
![Screenshot 2023-07-21 1 09 12 AM](https://github.com/dnschneid/crouton/assets/110459249/45d08a34-47ff-4d47-afae-867599e587ec)

**Comment by ghost** [comment]
Try `shell` before installing.
